[2020-09-29T11:26:53.016] [ERROR] errorLogger - 
*************** error log start ***************
request method: POST
request originalUrl:  /api/gm/game/creategame
request client ip:  ::ffff:127.0.0.1
request body: 
{"userId":"","gameName":"猪咬疯猴子","user":"1","gameid":"5"}
response time: 1601350013015
err name: SequelizeDatabaseError
err message: invalid input syntax for type integer: ""
err stack: SequelizeDatabaseError: invalid input syntax for type integer: ""
    at Query.formatError (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:366:16)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:72:18
From previous event:
    at Query.run (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:64:23)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:29
From previous event:
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:12
    at processImmediate (internal/timers.js:456:21)
From previous event:
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:651:10
    at H:\致命代码\fancyguo-gm\server\node_modules\retry-as-promised\index.js:70:21
    at new Promise (<anonymous>)
    at retryAsPromised (H:\致命代码\fancyguo-gm\server\node_modules\retry-as-promised\index.js:60:10)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:641:30
From previous event:
    at Sequelize.query (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:590:23)
    at _callee4$ (H:\致命代码\fancyguo-gm\server\src\app\service\/gm_game.js:60:31)
    at tryCatch (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
*************** error log end ***************

[2020-09-29T11:27:37.006] [ERROR] errorLogger - 
*************** error log start ***************
request method: POST
request originalUrl:  /api/gm/game/creategame
request client ip:  ::ffff:127.0.0.1
request body: 
{"userId":"","gameName":"1","user":"1","gameid":"5"}
response time: 1601350057005
err name: SequelizeDatabaseError
err message: invalid input syntax for type integer: ""
err stack: SequelizeDatabaseError: invalid input syntax for type integer: ""
    at Query.formatError (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:366:16)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:72:18
From previous event:
    at Query.run (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:64:23)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:29
From previous event:
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:12
    at processImmediate (internal/timers.js:456:21)
From previous event:
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:651:10
    at H:\致命代码\fancyguo-gm\server\node_modules\retry-as-promised\index.js:70:21
    at new Promise (<anonymous>)
    at retryAsPromised (H:\致命代码\fancyguo-gm\server\node_modules\retry-as-promised\index.js:60:10)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:641:30
From previous event:
    at Sequelize.query (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:590:23)
    at _callee4$ (H:\致命代码\fancyguo-gm\server\src\app\service\/gm_game.js:60:31)
    at tryCatch (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
*************** error log end ***************

[2020-09-29T11:30:08.758] [ERROR] errorLogger - 
*************** error log start ***************
request method: GET
request originalUrl:  /api/white/whiteFindName
request client ip:  ::ffff:127.0.0.1
request query:  {"gameid":"6"}
response time: 1601350208757
err name: TypeError
err message: Cannot read property '1' of undefined
err stack: TypeError: Cannot read property '1' of undefined
    at _callee3$ (H:\致命代码\fancyguo-gm\server\src\app\service\/white.js:66:21)
    at tryCatch (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
*************** error log end ***************

[2020-09-29T11:30:11.879] [ERROR] errorLogger - 
*************** error log start ***************
request method: GET
request originalUrl:  /api/white/whiteFindName
request client ip:  ::ffff:127.0.0.1
request query:  {"gameid":"6"}
response time: 1601350211879
err name: TypeError
err message: Cannot read property '1' of undefined
err stack: TypeError: Cannot read property '1' of undefined
    at _callee3$ (H:\致命代码\fancyguo-gm\server\src\app\service\/white.js:66:21)
    at tryCatch (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
*************** error log end ***************

[2020-09-29T11:31:32.999] [ERROR] errorLogger - 
*************** error log start ***************
request method: GET
request originalUrl:  /api/white/whiteFindName
request client ip:  ::ffff:127.0.0.1
request query:  {"gameid":"6"}
response time: 1601350292998
err name: TypeError
err message: Cannot read property '1' of undefined
err stack: TypeError: Cannot read property '1' of undefined
    at _callee3$ (H:\致命代码\fancyguo-gm\server\src\app\service\/white.js:67:21)
    at tryCatch (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
*************** error log end ***************

[2020-09-29T14:25:02.513] [ERROR] errorLogger - 
*************** error log start ***************
request method: GET
request originalUrl:  /api/user/permission?game=1&username=
request client ip:  ::ffff:127.0.0.1
request query:  {"game":"1","username":""}
response time: 1601360702512
err name: TypeError
err message: Cannot read property 'id' of undefined
err stack: TypeError: Cannot read property 'id' of undefined
    at _callee$ (H:\致命代码\fancyguo-gm\server\src\app\service\/user.js:50:12)
    at tryCatch (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
*************** error log end ***************

[2020-09-29T14:27:25.266] [ERROR] errorLogger - 
*************** error log start ***************
request method: GET
request originalUrl:  /api/user/permission?game=1&username=
request client ip:  ::ffff:127.0.0.1
request query:  {"game":"1","username":""}
response time: 1601360845265
err name: TypeError
err message: Cannot read property 'id' of undefined
err stack: TypeError: Cannot read property 'id' of undefined
    at _callee$ (H:\致命代码\fancyguo-gm\server\src\app\service\/user.js:50:12)
    at tryCatch (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
*************** error log end ***************

[2020-09-29T14:27:28.171] [ERROR] errorLogger - 
*************** error log start ***************
request method: GET
request originalUrl:  /api/user/permission?game=1&username=
request client ip:  ::ffff:127.0.0.1
request query:  {"game":"1","username":""}
response time: 1601360848171
err name: TypeError
err message: Cannot read property 'id' of undefined
err stack: TypeError: Cannot read property 'id' of undefined
    at _callee$ (H:\致命代码\fancyguo-gm\server\src\app\service\/user.js:50:12)
    at tryCatch (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
*************** error log end ***************

[2020-09-29T14:27:29.128] [ERROR] errorLogger - 
*************** error log start ***************
request method: GET
request originalUrl:  /api/user/permission?game=1&username=
request client ip:  ::ffff:127.0.0.1
request query:  {"game":"1","username":""}
response time: 1601360849127
err name: TypeError
err message: Cannot read property 'id' of undefined
err stack: TypeError: Cannot read property 'id' of undefined
    at _callee$ (H:\致命代码\fancyguo-gm\server\src\app\service\/user.js:50:12)
    at tryCatch (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
*************** error log end ***************

[2020-09-29T14:27:51.640] [ERROR] errorLogger - 
*************** error log start ***************
request method: GET
request originalUrl:  /api/user/permission?game=1&username=
request client ip:  ::ffff:127.0.0.1
request query:  {"game":"1","username":""}
response time: 1601360871640
err name: TypeError
err message: Cannot read property 'id' of undefined
err stack: TypeError: Cannot read property 'id' of undefined
    at _callee$ (H:\致命代码\fancyguo-gm\server\src\app\service\/user.js:50:12)
    at tryCatch (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
*************** error log end ***************

[2020-09-29T14:28:30.211] [ERROR] errorLogger - 
*************** error log start ***************
request method: GET
request originalUrl:  /api/user/permission?game=1&username=
request client ip:  ::ffff:127.0.0.1
request query:  {"game":"1","username":""}
response time: 1601360910210
err name: TypeError
err message: Cannot read property 'id' of undefined
err stack: TypeError: Cannot read property 'id' of undefined
    at _callee$ (H:\致命代码\fancyguo-gm\server\src\app\service\/user.js:50:12)
    at tryCatch (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
*************** error log end ***************

[2020-09-29T14:29:01.171] [ERROR] errorLogger - 
*************** error log start ***************
request method: GET
request originalUrl:  /api/user/permission?game=1&username=
request client ip:  ::ffff:127.0.0.1
request query:  {"game":"1","username":""}
response time: 1601360941170
err name: TypeError
err message: Cannot read property 'id' of undefined
err stack: TypeError: Cannot read property 'id' of undefined
    at _callee$ (H:\致命代码\fancyguo-gm\server\src\app\service\/user.js:50:12)
    at tryCatch (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
*************** error log end ***************

[2020-09-29T15:30:32.926] [ERROR] errorLogger - 
*************** error log start ***************
request method: GET
request originalUrl:  /api/user/permission?game=GM%E5%90%8E%E5%8F%B0&username=
request client ip:  ::ffff:127.0.0.1
request query:  {"game":"GM后台","username":""}
response time: 1601364632925
err name: SequelizeDatabaseError
err message: column "b.create_time" must appear in the GROUP BY clause or be used in an aggregate function
err stack: SequelizeDatabaseError: column "b.create_time" must appear in the GROUP BY clause or be used in an aggregate function
    at Query.formatError (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:366:16)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:72:18
From previous event:
    at Query.run (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:64:23)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:29
From previous event:
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:12
    at processImmediate (internal/timers.js:456:21)
From previous event:
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:651:10
    at H:\致命代码\fancyguo-gm\server\node_modules\retry-as-promised\index.js:70:21
    at new Promise (<anonymous>)
    at retryAsPromised (H:\致命代码\fancyguo-gm\server\node_modules\retry-as-promised\index.js:60:10)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:641:30
From previous event:
    at Sequelize.query (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:590:23)
    at _callee$ (H:\致命代码\fancyguo-gm\server\src\app\service\/user.js:66:39)
    at tryCatch (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
*************** error log end ***************

[2020-09-29T15:30:36.781] [ERROR] errorLogger - 
*************** error log start ***************
request method: GET
request originalUrl:  /api/user/permission?game=GM%E5%90%8E%E5%8F%B0&username=
request client ip:  ::ffff:127.0.0.1
request query:  {"game":"GM后台","username":""}
response time: 1601364636781
err name: SequelizeDatabaseError
err message: column "b.create_time" must appear in the GROUP BY clause or be used in an aggregate function
err stack: SequelizeDatabaseError: column "b.create_time" must appear in the GROUP BY clause or be used in an aggregate function
    at Query.formatError (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:366:16)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:72:18
From previous event:
    at Query.run (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:64:23)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:29
From previous event:
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:12
    at processImmediate (internal/timers.js:456:21)
From previous event:
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:651:10
    at H:\致命代码\fancyguo-gm\server\node_modules\retry-as-promised\index.js:70:21
    at new Promise (<anonymous>)
    at retryAsPromised (H:\致命代码\fancyguo-gm\server\node_modules\retry-as-promised\index.js:60:10)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:641:30
From previous event:
    at Sequelize.query (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:590:23)
    at _callee$ (H:\致命代码\fancyguo-gm\server\src\app\service\/user.js:66:39)
    at tryCatch (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
*************** error log end ***************

[2020-09-29T15:32:22.596] [ERROR] errorLogger - 
*************** error log start ***************
request method: GET
request originalUrl:  /api/user/permission?game=GM%E5%90%8E%E5%8F%B0&username=
request client ip:  ::ffff:127.0.0.1
request query:  {"game":"GM后台","username":""}
response time: 1601364742596
err name: SequelizeDatabaseError
err message: syntax error at or near "("
err stack: SequelizeDatabaseError: syntax error at or near "("
    at Query.formatError (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:366:16)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:72:18
From previous event:
    at Query.run (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:64:23)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:29
From previous event:
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:12
    at processImmediate (internal/timers.js:456:21)
From previous event:
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:651:10
    at H:\致命代码\fancyguo-gm\server\node_modules\retry-as-promised\index.js:70:21
    at new Promise (<anonymous>)
    at retryAsPromised (H:\致命代码\fancyguo-gm\server\node_modules\retry-as-promised\index.js:60:10)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:641:30
From previous event:
    at Sequelize.query (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:590:23)
    at _callee$ (H:\致命代码\fancyguo-gm\server\src\app\service\/user.js:67:39)
    at tryCatch (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
*************** error log end ***************

[2020-09-29T15:32:24.719] [ERROR] errorLogger - 
*************** error log start ***************
request method: GET
request originalUrl:  /api/user/permission?game=GM%E5%90%8E%E5%8F%B0&username=
request client ip:  
request query:  {"game":"GM后台","username":""}
response time: 1601364744718
err name: SequelizeDatabaseError
err message: syntax error at or near "("
err stack: SequelizeDatabaseError: syntax error at or near "("
    at Query.formatError (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:366:16)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:72:18
From previous event:
    at Query.run (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:64:23)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:29
From previous event:
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:12
    at processImmediate (internal/timers.js:456:21)
From previous event:
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:651:10
    at H:\致命代码\fancyguo-gm\server\node_modules\retry-as-promised\index.js:70:21
    at new Promise (<anonymous>)
    at retryAsPromised (H:\致命代码\fancyguo-gm\server\node_modules\retry-as-promised\index.js:60:10)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:641:30
From previous event:
    at Sequelize.query (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:590:23)
    at _callee$ (H:\致命代码\fancyguo-gm\server\src\app\service\/user.js:67:39)
    at tryCatch (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
*************** error log end ***************

[2020-09-29T19:18:54.710] [ERROR] errorLogger - 
*************** error log start ***************
request method: PUT
request originalUrl:  /api/gm/game/changeGame
request client ip:  ::ffff:127.0.0.1
request body: 
{"imageUrl":"blob:http://localhost/9d94235a-5bec-48f7-bca7-60d888909a61","gameNameChange":"","userIdChange":"","userId":"1","gameName":"空翼冒险团","id":"1","iamgeChange":"true","gameid":"5"}
response time: 1601378334709
err name: SequelizeDatabaseError
err message: syntax error at or near ":"
err stack: SequelizeDatabaseError: syntax error at or near ":"
    at Query.formatError (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:366:16)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:72:18
From previous event:
    at Query.run (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:64:23)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:29
From previous event:
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:12
    at processImmediate (internal/timers.js:456:21)
From previous event:
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:651:10
    at H:\致命代码\fancyguo-gm\server\node_modules\retry-as-promised\index.js:70:21
    at new Promise (<anonymous>)
    at retryAsPromised (H:\致命代码\fancyguo-gm\server\node_modules\retry-as-promised\index.js:60:10)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:641:30
From previous event:
    at Sequelize.query (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:590:23)
    at _callee6$ (H:\致命代码\fancyguo-gm\server\src\app\service\/gm_game.js:107:31)
    at tryCatch (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at gmGameService.<anonymous> (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at gmGameService.changeGame (H:\致命代码\fancyguo-gm\server\src\app\service\gm_game.js:301:28)
    at _callee5$ (H:\致命代码\fancyguo-gm\server\src\app\controllers\/gm_game.js:50:36)
    at tryCatch (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at changeGame (H:\致命代码\fancyguo-gm\server\src\app\controllers\gm_game.js:267:28)
    at dispatch (H:\致命代码\fancyguo-gm\server\node_modules\koa-compose\index.js:44:32)
    at next (H:\致命代码\fancyguo-gm\server\node_modules\koa-compose\index.js:45:18)
    at H:\致命代码\fancyguo-gm\server\node_modules\koa-router\lib\router.js:346:16
    at dispatch (H:\致命代码\fancyguo-gm\server\node_modules\koa-compose\index.js:44:32)
    at H:\致命代码\fancyguo-gm\server\node_modules\koa-compose\index.js:36:12
    at dispatch (H:\致命代码\fancyguo-gm\server\node_modules\koa-router\lib\router.js:351:31)
    at next (H:\致命代码\fancyguo-gm\server\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at _callee$ (H:\致命代码\fancyguo-gm\server\src\middleware\/index.js:48:11)
    at tryCatch (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at H:\致命代码\fancyguo-gm\server\src\middleware\/index.js:43:2
*************** error log end ***************

[2020-09-29T19:19:40.637] [ERROR] errorLogger - 
*************** error log start ***************
request method: PUT
request originalUrl:  /api/gm/game/changeGame
request client ip:  ::ffff:127.0.0.1
request body: 
{"imageUrl":"blob:http://localhost/1096ade1-42a3-497a-b658-6caa8fd1ee6f","gameNameChange":"","userIdChange":"","userId":"1","gameName":"空翼冒险团","id":"1","iamgeChange":"true","gameid":"5"}
response time: 1601378380635
err name: SequelizeDatabaseError
err message: syntax error at or near ":"
err stack: SequelizeDatabaseError: syntax error at or near ":"
    at Query.formatError (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:366:16)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:72:18
From previous event:
    at Query.run (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:64:23)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:29
From previous event:
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:12
    at processImmediate (internal/timers.js:456:21)
From previous event:
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:651:10
    at H:\致命代码\fancyguo-gm\server\node_modules\retry-as-promised\index.js:70:21
    at new Promise (<anonymous>)
    at retryAsPromised (H:\致命代码\fancyguo-gm\server\node_modules\retry-as-promised\index.js:60:10)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:641:30
From previous event:
    at Sequelize.query (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:590:23)
    at _callee7$ (H:\致命代码\fancyguo-gm\server\src\app\service\/gm_game.js:107:31)
    at tryCatch (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
*************** error log end ***************

[2020-09-29T19:22:43.940] [ERROR] errorLogger - 
*************** error log start ***************
request method: PUT
request originalUrl:  /api/gm/game/changeGame
request client ip:  ::ffff:127.0.0.1
request body: 
{"imageUrl":"http://106.75.7.83/images/game/1601378422864c1c45b2ff75acca5664a7984c503e63.jpg","gameNameChange":"true","userIdChange":"","userId":"1","gameName":"空翼冒险团2","file":"","id":"1","iamgeChange":"","gameid":"5"}
response time: 1601378563939
err name: SequelizeDatabaseError
err message: column "空翼冒险团2" does not exist
err stack: SequelizeDatabaseError: column "空翼冒险团2" does not exist
    at Query.formatError (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:366:16)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:72:18
From previous event:
    at Query.run (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\dialects\postgres\query.js:64:23)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:29
From previous event:
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:655:12
    at processImmediate (internal/timers.js:456:21)
From previous event:
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:651:10
    at H:\致命代码\fancyguo-gm\server\node_modules\retry-as-promised\index.js:70:21
    at new Promise (<anonymous>)
    at retryAsPromised (H:\致命代码\fancyguo-gm\server\node_modules\retry-as-promised\index.js:60:10)
    at H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:641:30
From previous event:
    at Sequelize.query (H:\致命代码\fancyguo-gm\server\node_modules\sequelize\lib\sequelize.js:590:23)
    at _callee7$ (H:\致命代码\fancyguo-gm\server\src\app\service\/gm_game.js:107:31)
    at tryCatch (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at gmGameService.<anonymous> (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at gmGameService.changeGame (H:\致命代码\fancyguo-gm\server\src\app\service\gm_game.js:343:28)
    at _callee5$ (H:\致命代码\fancyguo-gm\server\src\app\controllers\/gm_game.js:50:36)
    at tryCatch (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at changeGame (H:\致命代码\fancyguo-gm\server\src\app\controllers\gm_game.js:267:28)
    at dispatch (H:\致命代码\fancyguo-gm\server\node_modules\koa-compose\index.js:44:32)
    at next (H:\致命代码\fancyguo-gm\server\node_modules\koa-compose\index.js:45:18)
    at H:\致命代码\fancyguo-gm\server\node_modules\koa-router\lib\router.js:346:16
    at dispatch (H:\致命代码\fancyguo-gm\server\node_modules\koa-compose\index.js:44:32)
    at H:\致命代码\fancyguo-gm\server\node_modules\koa-compose\index.js:36:12
    at dispatch (H:\致命代码\fancyguo-gm\server\node_modules\koa-router\lib\router.js:351:31)
    at next (H:\致命代码\fancyguo-gm\server\node_modules\koa\node_modules\koa-compose\index.js:42:32)
    at _callee$ (H:\致命代码\fancyguo-gm\server\src\middleware\/index.js:48:11)
    at tryCatch (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:45:40)
    at Generator.invoke [as _invoke] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:281:22)
    at Generator.prototype.<computed> [as next] (H:\致命代码\fancyguo-gm\server\node_modules\regenerator-runtime\runtime.js:106:21)
    at asyncGeneratorStep (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at H:\致命代码\fancyguo-gm\server\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at H:\致命代码\fancyguo-gm\server\src\middleware\/index.js:43:2
*************** error log end ***************

